delegates.txt

```diff
@@ -176,6 +176,7 @@ Kevin Smith (KS)
 Kevin Venkiteswaran (KVN)
 Kirill Cherkoshin (KCN)
 Kris Gray (KGY)
+Kris Kowal (KK)
```

Comment:
We try to prefer three-letter shorthands now.

Reply:
Done